src/components/advanced-search/advanced-search-dates.vue

```diff
@@ -10,6 +10,7 @@
             v-model="startDateText"
             :max="currentDateString"
```

Comment:
Maybe this should be `endDateText` ?

Reply:
I don't think so, since the start date max by default is the current date.
The endDate is dynamic and its range is dictated by the start date.

Reply:
So the min end date is the current start date? If so, this is fine and this conversation can be resolved.

Reply:
Exactly :) 

![Screen Shot 2021-05-18 at 2 10 31 PM](https://user-images.githubusercontent.com/53542131/118723902-cfb26900-b7e2-11eb-9d12-285b432dbe5c.png)